components/EmbraceModal.vue

```diff
@@ -0,0 +1,176 @@
+        >
+          <div class="text-2xl font-bold grow shrink basis-auto max-md:max-w-full">
+            {{ embracePage?.data.preview_title || 'Email Preview' }}
+          </div>
+        </div>
+        <div class="self-stretch text-left text-xl tracking-wide mt-10 max-md:max-w-full max-md:mt-6">
+          <PrismicRichText
+            :field="embracePage?.data.preview_description"
+            class="text-md mb-4 whitespace-pre-wrap text-gray-800"
+            fallback="If you agree with the information below, click on 'Go To Your Mail App'. You may also edit the message body beforehand."
+          />
+        </div>
+        <div class="items-center self-stretch flex w-full justify-between gap-5 mt-6 max-md:max-w-full max-md:flex-wrap">
+          <div class="text-left text-xl font-medium leading-6 grow shrink basis-auto my-auto max-md:max-w-full">
+            {{ embracePage?.data.preview_area_label || 'Generated message' }}
+          </div>
+        </div>
+        <div
+          id="previewBox"
+          class="items-stretch self-stretch border border-[color:var(--brand-green-medium,#7BB123)] bg-white flex flex-col mt-4 p-2 rounded-2xl border-solid max-md:max-w-full max-md:px-5"
+        >
+          <div
+            name="addressFields"
+            class="justify-center items-stretch bg-slate-800 bg-opacity-10 flex flex-col px-3 py-2 rounded-lg max-md:max-w-full"
+          >
+            <EmailPreviewField
+              name="toField"
+              title="To: "
+              :value="form.bankEmail"
+            />
+            <EmailPreviewField
+              name="bccField"
+              title="BCC: "
+              :value=" embracePage?.data.bcc_email || 'embrace@bank.green'"
+            />
+            <EmailPreviewField
+              v-if="form?.email"
+              name="fromField"
+              title="From: "
+              :value="form.email"
+            />
+            <EmailPreviewField
+              name="subjectField"
+              title="Subject: "
+              :value="form?.subject || 'Error retrieving subject line'"
+            />
+          </div>
+          <TextField
+            v-model="messageBody"
+            class="col-span-full w-full text-gray-800"
+            rows="10"
+            name="embraceText"
+            type="text"
+            :value="form?.body"
+            dark
+          />
+        </div>
+        <div
+          class="button-white justify-center items-center mt-4 bg-white self-stretch flex gap-1 py-2 rounded-lg border border-gray-800 border-solid max-md:px-5"
+          @click.prevent.stop="copyText"
+        >
+          <img
+            loading="lazy"
+            src="https://cdn.builder.io/api/v1/image/assets/TEMP/d0e54027-9f1e-4d75-9014-63e737747321?"
+            class="aspect-square object-contain object-center w-6 overflow-hidden shrink-0 max-w-full"
+          >
+          <button
```

Comment:
this shouldn't be really necessary to have any HTML tag but just the text if we use a `button` tag as wrapper